front/app/components/Form/index.tsx

```diff
@@ -120,11 +120,22 @@ const Form = memo(
 
     const [apiErrors, setApiErrors] = useState<CLErrors | undefined>();
     const [loading, setLoading] = useState(false);
+    const [scrollToError, setScrollToError] = useState(false);
     const [showAllErrors, setShowAllErrors] = useState(false);
 
     const isSurvey = config === 'survey';
     const showSubmitButton = !isSurvey;
 
+    useEffect(() => {
```

Comment:
It would be better to use a ref for this - see how `scrollIntoView` is implemented in our main `UI/Error` component. Also wondering - would it be possible to use this main component here, instead of re-implementing it?

Reply:
Ah shoot - I missed that we had that already. I'll take a look :)

Reply:
@IvaKop Hmm the ErrorDisplay and Error components are very similar, but ErrorDisplay is handling the ajv form validation errors as well. I don't think I can replace it with just the Error component. I'd have to add ajv error handling into the Error component. But that is pretty specific for JSON forms, so I'm not sure if we want to do that.

Do you think I should keep it as is, but use the ref instead? :thinking:

Reply:
Yes, let's keep it as is for now - I'll just keep this in mind when we start working on improving error handling in the future.